### PR TITLE
Finished the capstone project

### PR DESCRIPTION
**Pull request by summyalena**
###  Hello Microverse,
_**I have successfully created the capstone project in partial fulfillment to all the requirements needed in the project.**_
- [ ]  I created a Youth summit conference website that has a basic home and about page.
- [ ] The youth summit conference is a website that creates an awareness for an upcoming program or event for a non-profit organization.
- [ ] - The speakers in the home page were dynamically created with Javascript objects.
- [ ] - I added some other features in the about page and ensured that it also follows the requirements for the project.

### CODE REVIEW

File: about.html [hunk 1]
```diff
@@ -0,0 +1,248 @@
...
+    <div class="hero-section-head about">
+          <h2>
+            <span class="gradient-text about">YOUTH HUMAN RIGHTS INTERNATIONAL SUMMIT 2022</span></h2>
+            <div class="about-line"></div>
+          <p class="p-about">Youth International is a non-profit organization that seeks to help youths align to their dreams and build them to be more enlightened on who they are and what they can bring. The Youth Summit 2022 is a conference organized with the aim of reaching out to as many youths for awareness.</p>
+          <p class="hero-p about"><span>please contact us per Email for further details about <br> YouthSummit 2015 <br><a href="summyalena@gmail.com" class="a"> youthsummit@gmail.com </a> </span></p>
+        </div>
+      </section>
+
+   <section class="main-program-about">
```
Comment (vechicin) [inline about.html:128]: You are missing an entire section.

Screen Shot 2022-07-28 at 8 47 24

I cannot see a section in your `About` page that fulfills the requirements above. Please add it.

File: README.md [hunk 2]
```diff
@@ -1 +1,84 @@
...
+
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+> This project objective is to understand how to build a portfolio, which is one of the most powerful tools in our software developer toolbox; it's the simplest way of demonstrating what we're truly capable of as developers, and it's a quick and easy way for recruiters and hiring managers to get an idea of what we can bring to their teams.
+
+## Description
+
+## Hosting
+
+This Portfolio is hosted on GitHub Page. [Click here]()
```
Comment (vechicin) [inline README.md]: Your live demo is not properly linked. Please fix it.

File: about.html [hunk 3]
```diff
@@ -0,0 +1,248 @@
+<!DOCTYPE html>
```
Comment (vechicin) [inline about.html:1]: Neither your `About` nor your `Home` page have an `<h1>` tag.

Screen Shot 2022-07-28 at 8 54 01

Please add them to both pages.

**Review by vechicin — CHANGES_REQUESTED** [review]
# Hi @summyalena ✋, 

Good job so far! 🔥 
There are some issues that you still need to work on to go to the next project but you are almost there! 🚨 
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights ⚡ 

- [x] Fantastic UI/UX!

## Required Changes ♻️

- [ ] Please attach the link to the video presentation either in your PR description or in the README file.

_Check the comments under the review._

## Optional suggestions 📂 
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Ghiftee — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️♻️
Hi there 👋🏽 , 

Great work on making the changes requested by a previous reviewer 👏 
To highlight 👍🏽 
- Your live link now works
- A video presentation has been added
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.



 ## Required Changes ♻️
- [ ] According to the requirements, it should be possible to navigate between the home and about page, however, when I click on the home hamburger icon or on the home button itself, I get a 404 error as shown below. Please fix the links.
Screenshot 2022-07-28 at 18 12 49

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by cynthiainga — APPROVED** [review]
Hi @summyalena, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


## Highlights

- [ ] All linters are passing :heavy_check_mark: 
- [ ] Good  design :100: 
- [ ] Good implementation of changes required :+1: 
- [ ] Descriptive PR :heavy_check_mark: 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**